editor/filesystem_dock.cpp

```diff
@@ -3261,9 +3266,27 @@ void FileSystemDock::_file_and_folders_fill_popup(PopupMenu *p_popup, const Vect
 	if (p_paths.size() == 1) {
 		const String &fpath = p_paths[0];
 
-#if !defined(ANDROID_ENABLED) && !defined(WEB_ENABLED)
 		p_popup->add_separator();
 
+		if (favorites_list.has(fpath)) {
+			TreeItem* favorites_item = tree->get_root()->get_first_child();
+			TreeItem* cursor_item = tree->get_selected();
```

Comment:
Style issue here: https://github.com/godotengine/godot/actions/runs/8833828638/job/24254083935?pr=91158

You can install the pre-commit hooks with `pip install pre-commit` and `pre-commit install` in the Godot repo, so it will run clang-format automatically on your commits.